### ISSUE TITLE
Julia cheatsheet from julialang.org is outdated 

### ISSUE THREAD

**Issue by iuliancioarca** [JuliaLang/www.julialang.org#646]
As described in:
https://discourse.julialang.org/t/julia-cheatsheet-from-julialang-org-is-outdated/35226
The following cheatsheet is missing some broadcasting dots in the "Arithmetic and functions of vectors and matrices" section. (last updated in Feb 2017):
https://github.com/mitmath/1806/blob/master/julia/Julia-cheatsheet.pdf

**Comment by inkydragon** [comment]
Maybe we can merge those content into: https://juliadocs.github.io/Julia-Cheat-Sheet/

**Comment by ViralBShah** [comment]
Perhaps remove it from this page to start with. Submit a PR?

**Comment by ViralBShah** [comment]
QuantEcon has very nice cheatsheets for Julia/Python/matlab : https://cheatsheets.quantecon.org/

**Comment by ViralBShah** [comment]
I don't think we have cheatsheets any more - so may be we should introduce these two links somewhere. Maybe on the "Getting Started with Julia" page.

**Comment by ChrisRackauckas** [comment]
I think we should just link the QuantEcon one. They spent a lot of time and it's really good.

**Comment by ViralBShah** [comment]
It should go here: https://julialang.org/learning/getting-started/

**Comment by ashwani-rathee** [comment]
We still need our own cheatsheet??or the QuantEcon(they are pretty good actually) would work??

**Comment by ViralBShah** [comment]
Use the existing ones. Here's a list the MIT class is using: https://computationalthinking.mit.edu/Fall20/cheatsheets/

**Comment by azev77** [comment]
1. here is a link to the updated MIT cheat sheets: https://computationalthinking.mit.edu/Spring21/cheatsheets/
2. the main Julia cheat sheet (https://juliadocs.github.io/Julia-Cheat-Sheet/) is dated. 
I made a few PRs but there's a ton of stuff. 
E.g. 
organization names are old (JuliaOpt etc) 
missing the last few years of JuliaCon (latest is 2018)
dead link to JuliaBox 
etc...
Can we have a community effort to update (https://juliadocs.github.io/Julia-Cheat-Sheet/) ?

**Comment by logankilpatrick** [comment]
We should probably move the cheatsheet from the JuliaDocs org and into the Julia Community org
